apps/angular/src/app/features/auth/login/login.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="header-container">
+	<h1 class="header-title">Welcome back</h1>
+	<p class="header-subtitle">Sign in to access to you dashboard.</p>
+</div>
+
+<form [formGroup]="loginForm" (ngSubmit)="onSubmit()" class="form">
+	<section class="form-control">
+		<label class="form__label">Email</label>
+		<mat-form-field subscriptSizing="dynamic" appearance="outline" class="form__input-field">
+			<mat-icon matIconPrefix>email</mat-icon>
+			<input matInput type="email" placeholder="Enter your email" [formControl]="loginForm.controls.email">
```

Comment:
You haven't connected label and input. Therefore, focusing on the input element will not work correctly, and the screenreader will not be able to read your input correctly.
You have 3 options for working with label + input:

1. Add a unique id for the input and use it in the label
2. Put input inside label
3. Use a ready-made element mat-label

```suggestion
		<mat-form-field subscriptSizing="dynamic" appearance="outline" class="form__input-field">
		<mat-label>Email</mat-label>
			<mat-icon matIconPrefix>email</mat-icon>
			<input matInput type="email" placeholder="Enter your email" [formControl]="loginForm.controls.email">
```